core/chains/evm/logpoller/log_poller_internal_test.go

```diff
@@ -71,36 +71,36 @@ func TestLogPoller_RegisterFilter(t *testing.T) {
 	require.Equal(t, 1, len(f.Addresses))
 	assert.Equal(t, common.HexToAddress("0x0000000000000000000000000000000000000000"), f.Addresses[0])
 
-	err := lp.RegisterFilter(Filter{"Emitter Log 1", []common.Hash{EmitterABI.Events["Log1"].ID}, []common.Address{a1}, 0})
+	err := lp.RegisterFilter(testutils.Context(t), Filter{"Emitter Log 1", []common.Hash{EmitterABI.Events["Log1"].ID}, []common.Address{a1}, 0})
```

Comment:
You've already assigned this to `ctx` so it's better if you pass that instead for less verbosity

Reply:
I agree. Think it'd be cleaner to define `ctx := testutils.Context(t)` at the top and pass just `ctx` for all of the methods below